Lib9c/Action/Buy.cs

```diff
@@ -24,7 +24,7 @@ namespace Nekoyume.Action
     /// </summary>
     [Serializable]
     [ActionType("buy12")]
-    public class Buy : GameAction, IBuy5
+    public class Buy : GameAction, IBuy5, IBuyV2
```

Comment:
The `IBuy5` seems introduced in https://github.com/planetarium/lib9c/pull/578 PR to be used in commands.

`IBuy5` provides a property with `IEnumerable<IPurchaseInfo>` type not `IEnumerable<IValue>`. In command, it doesn't need interfaces of `Lib9c.Abstractions` so I think two interfaces should be separated because their purposes are different.